pkgs/by-name/im/immich-machine-learning/package.nix

```diff
@@ -56,12 +57,14 @@ python.pkgs.buildPythonApplication rec {
     ]
     ++ uvicorn.optional-dependencies.standard;
 
-  nativeCheckInputs = with python.pkgs; [
-    httpx
-    pytest-asyncio
-    pytest-mock
-    pytestCheckHook
-  ];
+  nativeCheckInputs =
+    with python.pkgs;
+    [
+      httpx
+      pytest-asyncio
+      pytest-mock
+    ]
+    ++ (if stdenv.system == "aarch64-linux" then [ ] else [ pytestCheckHook ]);
```

Comment:
I'd appreciate using `lib.optionals` here:

```
lib.optionals (stdenv.system != "aarch64-linux") [ pytestCheckHook ]
```

But generally it might be better to set `doCheck = false` for aarch64-linux. See comment below.